src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -617,6 +617,29 @@ class LambdaTest extends ExternalServicesTestUtils with MockitoSugar {
       mockLambda.verifyInvocationsAndArgumentsPassed(folderIdsAndRows, 3)
     }
 
+  "handleRequest" should "call the DDB client's 'getAttributeValues' and entities client's 'entitiesByIdentifier' and " +
+    "'nodesFromEntity' method 2x but throw an exception if the parent ref of the top level folder is populated" in {
```

Comment:
1. We removed `nodesFromEntity`, if I recall, we get everything from `entitiesByIdentifier`, as well as removing from this test name, could you do it from others?
2. "top-level" instead of "top level"

Reply:
I've removed all the references to `nodesFromEntity` and added the hyphen.